docs/learn/learn-guides-treasury.md

```diff
@@ -109,6 +125,134 @@ in this document.
 The preimage and decision deposits
 [can be claimed once the referendum ends](./learn-guides-polkadot-opengov.md#claiming-the-preimage-and-decision-deposits).
 
+## Creating a USDT Treasury Proposal - Spend (with AssetHub)
+
+The following mostly goes over how to utilize the `spend` extrinsic, which unlike `spend_local`, is
+able to specify assets besides the native asset on system parachains, such as
+[AssetHub](../general/glossary.md#asset-hub).
+
+### Creating a preimage
+
+The example below shows how to create a [preimage](../general/glossary#preimage) for a transaction
+that requests 100 USDT from AssetHub.
+
+- Navigate to [Polkadot-JS UI > Governance > Referenda](https://polkadot.js.org/apps/#/preimages)
+  and then click on Add Preimage.
+- Select the account to submit the preimage.
+- Choose the`treasury` pallet in the "propose" dropdown and the
+  `spend(assetKind, amount, beneficiary, validFrom)` call
+
+Now, let's go by each field one-by-one and fill it out accordingly:
+
+### Specifying asset kind
+
+`assetKind` refers to the asset to be used, specified via XCM. In short, we need to be able to find:
+
+- Which system parachain the asset lives on
+- Which pallet and asset ID does it correspond to
+
+For this example, we are using USDT, which from the perspective of asking for USDT on the relay
+chain on AssetHub would be:
+
+`Parachain 1000 (AssetHub) > AssetId (Concrete) > PalletInstance 50 > General Index 1984`
+
+First, we specify which parachain - in this case, AssetHub (1000). `PalletInstance 50` refers to the
+asset pallet instance on AssetHub. The general index is `1984`, which is the ID of USDT on the asset
+pallet on AssetHub.
+
+Here is how the final `assetKind` field should look:
+
+### Specifying the amount
+
+The amount should be simply the amount of USDT, where each `1` USDT is `1000000`. Because we are
+asking for 100 USDT, we put `100000000` as the input for the amount:
```

Comment:
Based on the `:` I assume you intend to put screenshots here?

Reply:
Yes, will presumably be done after the written content is finalized